app/build.gradle

```diff
@@ -116,6 +116,10 @@ android {
             applicationIdSuffix ".debug"
             versionNameSuffix "-debug"
             signingConfig signingConfigs.debug
+
+            buildConfigField "String", "IP_SIMPLE_API_ACCESS_KEY", "\"AIzaSyBhJJeiBWKTfbV446m2sgJCP-EogEWZUh0\""
```

Comment:
Should we use the google api gradle plugin an a json file?


[image: --]
Friedger Müffke
[image: http://]about.me/friedger
<http://about.me/friedger>


2015-09-30 16:17 GMT+02:00 Said Tahsin Dane <notifications@github.com>:

> In app/build.gradle
> <https://github.com/gdg-x/frisbee/pull/490#discussion_r40798422>:
>
> > @@ -116,6 +116,10 @@ android {
> >              applicationIdSuffix ".debug"
> >              versionNameSuffix "-debug"
> >              signingConfig signingConfigs.debug
> > +
> > +            buildConfigField "String", "IP_SIMPLE_API_ACCESS_KEY", "\"AIzaSyBhJJeiBWKTfbV446m2sgJCP-EogEWZUh0\""
>
> This part is the debug buildType where arbitrary but valid keys are used.
> Should I move these to fields?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/490/files#r40798422>.
>